src/requirements/data/majors/comm.ts

```diff
@@ -0,0 +1,156 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { courseIsFWS, includesWithSingleRequirement, includesWithSubRequirements } from '../checkers-common';
+
+const commRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Core Courses',
+    description: '15 credits in COMM 1101, COMM 1300, COMM 2010, COMM 2310, COMM 2820',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSubRequirements(
+      ['COMM 1101'],
+      ['COMM 1300'],
+      ['COMM 2010'],
+      ['COMM 2310'],
+      ['COMM 2820']
+    ),
+    operator: 'and',
+    fulfilledBy: 'credits',
+    minCount: 15
+  },
+  {
+    name: 'Focus Area',
+    description: '6 credits in COMM 2200, COMM 2450, COMM 2760, or COMM 2850',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSubRequirements(
+      ['COMM 2200'],
+      ['COMM 2450'],
+      ['COMM 2760'],
+      ['COMM 2850'],
+    ),
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 6
+  },
+  {
+    name: 'Focus Area Upper Level',
+    description: 'Students must take six credits (two courses) of coursework within their declared Focus Area at the 3100+ level.',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSingleRequirement(
+      'COMM 31**',
```

Comment:
i think ur missing COMM 2179, MUSIC/PMA/COML/ENG 2703

Reply:
wait actually doing the *** includes classes that shouldn't be. for example, COMM 4350 counts now when it shouldn't.